openedx/core/djangoapps/oauth_dispatch/views.py

```diff
@@ -5,21 +5,17 @@
 
 from __future__ import unicode_literals
 
-import hashlib
```

Comment:
FYI: I think the import clean-up is left over from the earlier PR where we deleted some views.